demo/src/examples/examples-page.js

```diff
@@ -0,0 +1,76 @@
+import router from '../router/router';
+import { css, html, LitElement } from 'lit';
+import { animations, theme } from '../theme/theme';
+import './load-media-form-component';
+import Examples from './examples';
+import { openPlayerModal } from '../player/player-dialog';
+import { map } from 'lit/directives/map.js';
+import { when } from 'lit/directives/when.js';
+
+
+export class ExamplesPage extends LitElement {
+  static styles = [
+    theme, animations, css`
+      .example-section p {
+        margin-bottom: 0;
+        color: var(--color-5);
+        font-size: var(--size-3);
+        text-align: left;
+      }`
+  ];
+
+  constructor() {
+    super();
+  }
```

Comment:
Refer to https://github.com/SRGSSR/pillarbox-web/pull/141/files#r1429898660

Reply:
Removed.